src/Qowaiv.DomainModel/AggregateRoot_TAggregate.cs

```diff
@@ -78,15 +81,22 @@ protected Result<TAggregate> Apply(IEnumerable<object> events)
     /// <summary>Loads the state of the aggregate root by replaying events.</summary>
     protected void Replay(IEnumerable<object> events)
     {
-        foreach (var @event in (events ?? Array.Empty<object>()).Where(IsSupported))
+        events ??= Array.Empty<object>();
```

Comment:
Instead, you could just return when `events` is `null`. Since you technically support `null`, you could also consider changing the contract to accept `null` values.

Reply:
It is accepted, as a defensive measure, but it should not be encouraged.